src/plugins/intel_gpu/include/intel_gpu/runtime/layout.hpp

```diff
@@ -306,9 +315,12 @@ struct layout {
         return (lhs.data_padding < rhs.data_padding);
     }
 
-    /// Number of elements to be stored in this memory layout
+    /// Number of elements to be stored in this layout
     size_t count() const;
 
+    /// Number of elements to be stored in this memory layout including padding
+    size_t padded_count() const;
```

Comment:
btw, looks like this method duplicates `get_linear_size` functionality. Maybe we can use latter one everywhere?

Reply:
Updated